bundle/config/validate/for_each_task_test.go

```diff
@@ -0,0 +1,191 @@
+package validate
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config"
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/bundle/internal/bundletest"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func createBundleWithForEachTask(parentTask jobs.Task) *bundle.Bundle {
+	if parentTask.ForEachTask == nil {
+		parentTask.ForEachTask = &jobs.ForEachTask{
+			Inputs: "[1, 2, 3]",
+			Task: jobs.Task{
+				TaskKey: "child_task",
+				NotebookTask: &jobs.NotebookTask{
+					NotebookPath: "test.py",
+				},
+			},
+		}
+	}
+
+	b := &bundle.Bundle{
+		Config: config.Root{
+			Resources: config.Resources{
+				Jobs: map[string]*resources.Job{
+					"job1": {
+						JobSettings: jobs.JobSettings{
+							Name:  "My Job",
+							Tasks: []jobs.Task{parentTask},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	bundletest.SetLocation(b, "resources.jobs.job1.tasks[0]", []dyn.Location{{File: "job.yml", Line: 1, Column: 1}})
+	return b
+}
+
+func TestForEachTask_InvalidRetryFields(t *testing.T) {
```

Comment:
The test looks good overall, but generally for testing mutators, we prefer to use acceptance tests (see `acceptance` folder). Could you change this test to an acceptance test instead?